src/inject/settings/tray.js

```diff
@@ -10,3 +10,20 @@ if (Settings.get('minToTray', true)) {
 } else {
   $('#min-to-tray').removeAttr('checked');
 }
+
+$('#show-tray').change((e) => {
```

Comment:
It looks like you haven't thought through the implications of this for all platforms.  For instance it appears at the moment on Windows that I can untick "Show in Tray" but still have "Minimize to Tray" ticked but disabled.

This means I now have NO way to quit the application.  It would just stay open always :-1: